packages/stripe/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 11.6.0
```

Comment:
We will define the version (I think we want to add some more) to this can be removed